google-cloud-bigtable/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Release History
 
+### 0.8.0 / 2019-11-01
+
+#### ⚠ BREAKING CHANGES
+
+* **bigtable:** The following methods now raise Google::Cloud::Error instead of
```

Comment:
```suggestion
* The following methods now raise Google::Cloud::Error instead of
  Google::Gax::GaxError and/or GRPC::BadStatus:
  * Table#mutate_row
  * Table#read_modify_write_row
  * Table#check_and_mutate_row
  * Table#sample_row_keys
```

Reply:
SGTM, I'll also apply the list in the features section below.